tests/gui/test_main.py

```diff
@@ -38,11 +38,12 @@ def test_init(mocker):
     load_css = mocker.patch("securedrop_client.gui.main.load_css")
 
     app_state = state.State()
-    w = Window(app_state)
+    export_service = export.Service()
+    w = Window(app_state, export_service)
```

Comment:
Note how this unit test uses a real `export.Service` instance :slightly_smiling_face: